db/seeds/activities.rb

```diff
@@ -38,7 +38,8 @@
 
     location:          Faker::TvShows::FamilyGuy.location,
     organized_by:      Faker::Boolean.boolean(true_ratio: 0.8) ? Group.all.sample : nil,
-    description:       Faker::Lorem.paragraph(sentence_count: 5),
+    description_nl:    Faker::Lorem.paragraph(sentence_count: 5),
```

Comment:
Faker can generate multilingual texts afaik

Reply:
So I've tried a lot of different Faker types and in the end I found the [NL locale file](https://github.com/faker-ruby/faker/blob/master/lib/locales/nl.yml). It turns out they do not have a lot of Dutch translations, so I think we shoudl stick to the Lorem text

Reply:
makes sense, thanks for looking into it